src/runtime_lib/infra_ligra/ligra/parallel.h

```diff
@@ -50,10 +90,50 @@ static void setWorkers(int n) {
 // intel cilk+
 #elif defined(CILKP)
```

Comment:
Why do we need a CILKP in addition to CILK flag?

Reply:
Hmm I think this existed before the TBB PR. Maybe some legacy stuff?